site/gatsby-site/migrations/2022.06.07T19.46.03.cloudinary-id-update.js

```diff
@@ -18,7 +18,7 @@ const updateCollection = async (collection) => {
   while (await cursor.hasNext()) {
     const document = await cursor.next();
 
-    const cloudinary_id = getCloudinaryPublicID(document.image_url);
+    const cloudinary_id = getCloudinaryPublicID(document.media_url);
```

Comment:
Two issues here. 1. Don't change historical migrations. 2. This orphans all the data stored at `image_url`. You need to migrate the values by setting the new value and unsetting the old value. This won't be mergeable without that.

Reply:
Yes. When you change a historically applied migration, you are breaking the history of changes and the databases will be in a more uncertain state.